src/results/result-form.resource.ts

```diff
@@ -19,6 +19,12 @@ export interface ConceptResponse {
   attributes: any[];
   links: Link18[];
   resourceVersion: string;
+  hiNormal: number;
+  hiAbsolute: number;
+  hiCritical: number;
+  lowNormal: number;
+  lowAbsolute: number;
+  lowCritical: number;
```

Comment:
Are these generic to all concept responses? if not we should make them ternary 

```suggestion
  hiNormal?: number;
  hiAbsolute?: number;
  hiCritical?: number;
  lowNormal?: number;
  lowAbsolute?: number;
  lowCritical?: number;
```